src/Other/Tokenizers/SkipGram.php

```diff
@@ -81,12 +81,6 @@ public function __construct(int $n = 2, int $skip = 2, ?Word $wordTokenizer = nu
         $this->wordTokenizer = $wordTokenizer ?? new Word();
     }
 
-    /**
-     * Tokenize a blob of text.
-     *
-     * @param string $string
-     * @return string[]
```

Comment:
Each method requires a docblock

Reply:
won't this now inherit the docblock defined in the interface `Rubix\ML\Other\Tokenizers::tokenize()`, or do you need an explicit 

```php
/**
 * {@inheritdoc}
 */
```

Reply:
I'm not familiar with {@inheritdoc} but I use these docblocks as a summary of the inputs and outputs of the method. It looks like that information just vanished.

Reply:
Btw we don't use PHPdoc, I use those docblocks for my own reference

https://docs.phpdoc.org/latest/guides/inheritance.html

Reply:
We don't use PHPdoc and I use these docblocks as a reference when I'm programming. If we delete them and put {@inheritdoc} instead then, to me, the information just vanishes. Am I doing it wrong?

Reply:
I expanded as to why the removal of these non-essential docblocks is actually meaningful for static analysis in this scenario in this comment: https://github.com/RubixML/RubixML/pull/114#discussion_r492344856

I'm not advocating for the use of inheritdoc tags, but if the coding standard enforces a docblock then `{@inheritdoc}` would satisfy that condition whilst also introducing increased type-safety.

Reply:
I still don't get it ... considering I look at these docblocks when I'm programming to understand the inputs and outputs, types, and general purpose of the thing - how do I get that info into my brain/short-term memory if all I see is "{@inheritdoc}" in my IDE? It seems that I would have to go hunting for the information that I would normally have up-front. Am I missing something?

Reply:
Yep. I agree, I too often naturally to use docblock over IDE magic. 

My only reason to remove this docblock as part of this PR is to allow the `@return list<string>` annotation I added to the `Tokenizer` interface to be used by static analyzers like phpstan/psalm:

docblocks naturally inherit from their parent for machines - (parsers use `ReflectionClass::getDocComment()` that naturally traverses to the parent definition). This is why the additional type-safety of introducing `list<string>` is only meaningful when static analyzers can access those annotations. (whilst most projects don't use phpdoc for documentation the phpdoc standard is used by static analyzers to read the annotations added to code) Removing this docblock means that static analysis will now default to the docblock that is available in the interface.

I wasn't really advocating for including `{@inheritdoc}` (personally, I don't like them) I was more suggesting a solution that allowed for this method to have a docblock, but an also inherit the improvement of annotating the return type as a `list<string>`, In previous PRs you have shown an aversion for non-essential/self-evident comments so was incorrectly pre-empting your preference here 🤣.

We could equally just retain these docblocks but _also_ update their return type annotations to be `list<string>`, (or, investigate a `php-cs-fixer` configuration that will enforce this standard and automatically add these in if missing)

Reply:
I wasn't making any statements in the above comment ... I am genuinely confused as to what the purpose/benefit of docblock inheritance is

Reply:
Gotcha - I guess inheritdoc just tries to apply a DRY convention to avoid having to update docblocks all over a codebase instead of just on the parent: but it comes at the significant expense of human readability. I'll add them back in with updated return types. 

Anecdotally, my IDE compiles them and displays them like this:

![PHPStorm - Screenshot 09:39am - chriss@local](https://user-images.githubusercontent.com/520892/93860164-90b12f80-fcb6-11ea-931f-30237aeb004f.png)


This little discussion here got me thinking about how docblocks inhabit an awkward space between 'comments' and 'code' (they contain both arbitrary comments aimed at humans AND also meaningful annotations/code aimed at machines) Once annotations [[rfc:accepted](https://wiki.php.net/rfc/shorter_attribute_syntax_change)] land in PHP8 annotations will be supported directly at language level and not have to be parsed out of comment blocks, so maybe clearer conventions will emerge then (Maybe: include specific static analysis metadata on the parent using annotations, liberally include helpful explanation in the doc comments wherever you can) 👍